pkgs/desktops/gnome-3/apps/gnome-power-manager/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, intltool
+, fetchurl
+, pkgconfig
+, gtk3
+, glib
+, meson
+, ninja
+, upower
+, desktop_file_utils
+, wrapGAppsHook
+, gnome3 }:
+
+stdenv.mkDerivation rec {
+  inherit (import ./src.nix fetchurl) name src;
+
+  propagatedUserEnvPkgs = [ gnome3.gnome_themes_standard ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    wrapGAppsHook
+
+    # needed by meson_post_install.sh
+    glib.dev
+    desktop_file_utils
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    upower
+    intltool
+    gnome3.defaultIconTheme
+    gnome3.gsettings_desktop_schemas
```

Comment:
It does not look like the schemas are used.

Reply:
The app creates it's own gsettings:  https://github.com/GNOME/gnome-power-manager/blob/9c91bf6ed47265466f06677fddb1940bebfd6e9e/src/gpm-statistics.c#L1923

What does `gnome3.gesttings_desktop_schemas` do?  Does it provides some default gsettings?